src/openrct2/world/Duck.cpp

```diff
@@ -20,14 +20,6 @@
 #include <limits>
 
 // clang-format off
```

Comment:
Move this to line 24, please

Reply:
After moving, the code did not pass the clang-format check.

Reply:
Seems pretty simple to solve, just delete an extra line there

Reply:
Thank you, modified.